crates/gpio/src/backend.rs

```diff
@@ -86,8 +100,16 @@ impl TryFrom<&str> for DeviceConfig {
         let mut devices = DeviceConfig::new();
 
         for info in list.iter() {
-            let number = info.parse::<u32>().map_err(Error::ParseFailure)?;
-            devices.push(number)?;
+            devices.push(match info.strip_prefix('s') {
```

Comment:
Maybe, we can implement new() for GpioDeviceConfig for and move the entire match block into that, which will make this just:

devices.push(GpioDeviceConfig::new(info)?)?;

Reply:
OK, I've wrapped it up in new().